scripts/update-appup.sh

```diff
@@ -77,14 +80,17 @@ fi
 
 PREV_DIR_BASE="/tmp/emqx-appup-build"
 mkdir -p "${PREV_DIR_BASE}"
-if [ ! -d "${PREV_TAG}" ]; then
+if [ ! -d "${PREV_DIR_BASE}/${PREV_TAG}" ]; then
     cp -R . "${PREV_DIR_BASE}/${PREV_TAG}"
 fi
 
 pushd "${PREV_DIR_BASE}/${PREV_TAG}"
+REMOTE="$(git remote -v | grep "${GIT_REPO}" | head -1 | awk '{print $1}')"
 git reset --hard
-git checkout "${PREV_TAG}"
+git clean -fxxx
```

Comment:
Q: does using multiple `-x` change the behavior?

Reply:
no, i'll fix it.